docs/documentation/helm.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Helm
+description: Helm
+layout: docs
+permalink: /docs/helm
+---
+
+### Helm
+
+Dekorate also supports generating manifests for `helm`. To make use of this feature you need to add:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>helm-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+This module provides the [@HelmChart](annotations/helm-annotations/src/main/java/io/dekorate/helm/annotation/HelmChart.java) that will generate the following Helm resources:
```

Comment:
Is it the proper wording to be used. Do we name all the features (Kubernetes, Docker, Knative, ...) of Dekorate as `modules` ?

Remark: I think that we should rephrase the beginning of this chapter and start with:
- First introduce helm dekorate
- Next, explain what you must do to generate the Helm manifests
- ...

Reply:
Changed